tests/containers/buildah.pm

```diff
@@ -25,7 +25,7 @@ use Utils::Backends qw(is_svirt);
 sub run_tests {
     my $runtime = shift;
 
-    my $image = "registry.opensuse.org/opensuse/tumbleweed:latest";
+    my $image = get_var("CONTAINER_IMAGE_TO_TEST", "registry.opensuse.org/opensuse/tumbleweed:latest");
     record_info('Test', "Pull image $image");
     assert_script_run("buildah pull $image", timeout => 300);
     validate_script_output('buildah images', sub { /registry.opensuse.org\/opensuse\/tumbleweed/ });
```

Comment:
Not sure whether this is a good change. If the container will be non-tw `buildah images` will fail as it expects only tw image

Reply:
Fixed.